services/src/main/scala/cromwell/services/metadata/impl/MetadataDatabaseAccess.scala

```diff
@@ -78,10 +78,8 @@ trait MetadataDatabaseAccess {
       val value = metadataEvent.value map { _.value }
       val valueType = metadataEvent.value map { _.valueType.typeName }
       val jobKey = key.jobKey map { jk => (jk.callFqn, jk.index, jk.attempt) }
-      MetadataEntry(
-        workflowUuid, jobKey.map(_._1), jobKey.flatMap(_._2), jobKey.map(_._3), key.key, value, valueType, timestamp)
+      MetadataEntry(workflowUuid, jobKey.map(_._1), jobKey.flatMap(_._2), jobKey.map(_._3), key.key, value, valueType, timestamp)
```

Comment:
ToL: Why do we bother exploding out the jobkey aboe? It seems like it then forces us to use the horrible `_._1` syntax on this line when we could instead say:
```
MetadataEntry(workflowUuid, key.jobKey.map(_.callFqn), key.jobKey.flatMap(_.index), key.jobKey.map(_.attempt), key.key, value, valueType, timestamp)
```